### PR TITLE
Add words and java/scala package naming formatting

### PR DESCRIPTION
**Pull request by gheine**

### CODE REVIEW

**Comment by gheine** [comment]
@xuwei-k do you think this could be merged?

**Comment by gheine** [comment]
Is this repository still being maintained? Anyone know if there's a way to get sbt use a fork of giter8 rather than the foundweeks release?

File: docs/03/a.md [hunk 1]
```diff
@@ -18,11 +18,13 @@ The formatting options are:
     decap    | decapitalize    : lowercase first letter
     start    | start-case      : uppercase the first letter of each word
     word     | word-only       : remove all non-word letters (only a-zA-Z0-9_)
+    words    | words-only      : replace all non-word letters (only a-zA-Z0-9) with a whitespace
```
Comment (eed3si9n) [inline docs/03/a.md]: I think `word` vs `words` is too subtle. How about `space`?

Comment (gheine) [reply docs/03/a.md]: that's fair. what about `sentence`?

Comment (eed3si9n) [reply docs/03/a.md]: Doesn't sentence imply some sort of grammatical structure? Turning "My-Example-Project" into "My Example Project" is not exactly sentencifying things.

**Comment by gheine** [comment]
@eed3si9n done

**Pull request merged**